index.html

```diff
@@ -16,9 +16,21 @@
             <div class="logo"><a href="#home"><p class="logo-txt poppins">My Logo</p></a></div>
             <nav class="menu"><div><img class="menu-im" alt="Menu" width="48px" height="48px" src="images/menu1.png"></div></nav>
         </div>
+        <nav>
+            <div class="container-5">
+                <div class="logo"><a href="#home"><p class="logo-txt poppins">My Logo</p></a></div>
+                <div class="">
+                    <ul class="nav-menu">
+                        <li><a class="poppins nav-menu-text" href="#portfolio">Portfolio</a></li>
+                        <li><a class="poppins nav-menu-text" href="#about-me">About</a></li>
+                        <li><a class="poppins nav-menu-text" href="#contact-form">Contact</a></li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
     </header>
     <section class="headline">
-        <h1 class="name poppins">I'm Juan Carlos. Glad to see you!</h1>
+        <h1 id="delay" class="name poppins">I'm Juan Carlos. Glad to see you!</h1>
         <p class="desc poppins">I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
```

Comment:
- [ ] Please consider adding All the decoration images, currently, your design does not show any decoration images. Remember as a frontend engineers we are expected to deliver almost identical webpages. 
### Your design as of now,
![image](https://user-images.githubusercontent.com/69077061/230749726-f59c666a-b385-4e11-9f62-43968bb57e96.png)